src/server/req_delete.c

```diff
@@ -174,6 +180,7 @@ job *pjob;
 			free_br(preq);
 		}
 	}
+	return (rc);
```

Comment:
Remove the ().  I checked and other functions in this file don't seem to use them for return.